go-controller/pkg/cluster/cluster.go

```diff
@@ -34,7 +34,14 @@ const (
 	// OvnHostSubnet is the constant string representing the annotation key
 	OvnHostSubnet = "ovn_host_subnet"
 	// OvnClusterRouter is the name of the distributed router
-	OvnClusterRouter = "ovn_cluster_router"
+	OvnClusterRouter              = "ovn_cluster_router"
+	OvnNodeGatewayMode            = "ovn_node_gateway_mode"
+	OvnNodeVlanID                 = "ovn_node_vlan_id"
+	OvnNodeIfaceID                = "ovn_node_iface_id"
+	OvnNodeMacAddress             = "ovn_node_mac_address"
+	OvnNodeSubnet                 = "ovn_node_subnet"
+	OvnNodeLocalnetGatewayIP      = "ovn_node_localnet_gateway_ip"
+	OvnNodeLocalnetGatewayNextHop = "ovn_node_localnet_gateway_next_hop"
```

Comment:
For these two, we want to make them generic since we're doing this not just for localnet gateway. SO perhaps:

`OvnNodeGatewayInterfaceIP`
`OvnNodeGatewayInterfaceNextHop`